src/ci/build_tools.py

```diff
@@ -314,19 +341,19 @@ def makeLib(moduleName):
     Example:
         makeLib("syscheckd")
     """
-    command = "make -C {}".format(utils.moduleDirPathBuild(moduleName))
+    command = "make -C {} -j4".format(utils.moduleDirPathBuild(moduleName))
```

Comment:
Maybe the jobs can be set or overwritten by parameter

Reply:
A new option for the script was added.